Makefile

```diff
@@ -0,0 +1,41 @@
+
+GO_BUILD_ENV :=
+GO_BUILD_FLAGS :=
+MODULE_BINARY := bin/obstacles-distance
+
+ifeq ($(VIAM_TARGET_OS), windows)
+	GO_BUILD_ENV += GOOS=windows GOARCH=amd64
+	GO_BUILD_FLAGS := -tags no_cgo	
+	MODULE_BINARY = bin/obstacles-distance.exe
+endif
+
+$(MODULE_BINARY): Makefile go.mod *.go cmd/module/*.go 
+	$(GO_BUILD_ENV) go build $(GO_BUILD_FLAGS) -o $(MODULE_BINARY) cmd/module/main.go
+
+lint:
+	gofmt -s -w .
+
+update:
+	go get go.viam.com/rdk@latest
+	go mod tidy
+
+test:
+	go test ./...
+
+module.tar.gz: meta.json $(MODULE_BINARY)
+ifeq ($(VIAM_TARGET_OS), windows)
+	jq '.entrypoint = "./bin/obstacles-distance.exe"' meta.json > temp.json && mv temp.json meta.json
```

Comment:
Leave this as-is, but FYI this won't be generated again after next week's release, per https://github.com/viamrobotics/rdk/pull/5063